src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -170,9 +181,10 @@ const AuthenticatorButton: UISchemaElementComponent<{
             paragraph
             id={`${iconName}-description`}
             sx={{
-              fontSize: '.875rem',
+              fontSize: tokens.TypographySizeSubordinate,
+              color: tokens.TypographyColorSubordinate,
               margin: 0,
-              marginBlockEnd: '6px',
+              marginBlockStart: '6px',
```

Comment:
Can we update all spacing styling (margins, padding, etc.) to use the closest `tokens.SpacingX` value? I believe we are planning on updating the Okta-hosted page to match Odyssey's expected base font size of `14px` so that would mean `tokens.Spacing1 = 4px` and `tokens.Spacing2 = 8px`. I'd say try those values out and pick the one that looks the best

See: https://github.com/okta/odyssey/blob/main/packages/odyssey-design-tokens/src/spacing.json

Reply:
Actually, I'd like @lesterchoi-okta 's thoughts on whether we want to update spacing values to use tokens as part of the migration. I could see an argument that we don't want this spacing to change based on tokens? I just looked at a Slack thread we had with Odyssey and they were thinking of keeping non-type-related spacing values as hard-coded `px`, see: https://okta.slack.com/archives/C7T2H3KNJ/p1701230964122719?thread_ts=1701193300.526959&cid=C7T2H3KNJ